src/test/groovy/com/heroku/sdk/gradle/HerokuPluginTest.groovy

```diff
@@ -185,12 +184,12 @@ class HerokuPluginTest extends Specification {
 
         then:
         buildResult.task(':deployHeroku').outcome == TaskOutcome.SUCCESS
-        buildResult.output.contains("app: ${ appName }")
+        buildResult.output.contains(appName)
         buildResult.output.contains("including: build/")
         buildResult.output.contains("- success")
         buildResult.output.contains("Installing JDK 1.8")
         buildResult.output.contains("Done")
 
-        exec("heroku run cat ${exec("pwd").trim()}README.md -a ${appName}").contains("README.md")
+        exec("heroku run ls -a ${appName}").contains("README.md")
```

Comment:
The old test was moot since `heroku run` will log the command it executes which contains the string `README.md`. This resulted in this test always passing and not testing if the file was actually included.